src/main/webapp/app/trials/views/trial.questionnaire.view.htm

```diff
@@ -18,31 +18,31 @@
 
 
 
-                    <div class="answers" ng-if="questionnaireEntry.answers._embeddedItems.length<=4">
+                    <div class="answers" data-ng-if="questionnaireEntry.answers._embeddedItems.length<=4">
                         <label data-ng-repeat="answer in questionnaireEntry.answers._embeddedItems | orderBy:'id'">
```

Comment:
orderBy:'sortOrder' with the new member available. 

Now that the sortOrder is available, when a user creates a new answer, we'll need to initialise the client side created answer object say with a sortOrder member equal to the length of the existing array of answers so that it naturally appears furthermost right before the save.